README.md

```diff
@@ -99,6 +100,8 @@ Don't forget to clean up if you're done kicking the tires:
 opstrace destroy aws <choose_a_name>
 ```
 
+Note: if you run into trouble cleaning up after a failed install, add the `--region` flag to ensure the CLI can locate any dangling resources.
```

Comment:
Maybe we should remove the part about `if you run into trouble cleaning up after a failed install` since this could also apply if a user exited out of the destroy command after the k8s cluster was destroyed, but before completion.

I'd actually advocate for adding the --region <region> to the default command in these docs (however one is supposed to document optional parameters):
```
opstrace destroy aws <choose_a_name> [--region <region>]
```

Reply:
I can remove it. My thinking was to just add some additional help for folks the way things work _right now_.  I'll remove it for now and we can discuss how to handle this offline.

Reply:
I think having that additional help is good - I'd just make it even more prominent (i.e. with less text before you get to the "add the --region flag" part. We already know that people don't read that far (because it's printed in the CLI and still gets missed)

Reply:
Ok, I've updated it—PTAL.